libcodechecker/util.py

```diff
@@ -590,3 +590,24 @@ def trim_path_prefixes(path, prefixes):
         return path
 
     return path[len(longest_matching_prefix):]
+
+
+def get_report_path_hash(report, files):
```

Comment:
I think it would be better to put this into `libcodechecker/report.py` so every report related function is there. It could be a static method of the Report class there.